templates/macros/filters/date.html

```diff
@@ -5,20 +5,16 @@
     <legend for="{{ name }}" class="label">{{ label }}</legend>
```

Comment:
This would change all the date filters, and not just the receipts and disbursements. This filter works well for filings. (If it is too annoying to have both filters now, it can wait until after launch)